cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/SlangBuildMain.java

```diff
@@ -91,6 +93,20 @@ public static void main(String[] args) {
         }
     }
 
+    private static List<String> parseTestSuites(ApplicationArgs appArgs) {
+        boolean runDefaultTests = true;
+        List<String> testSuites = ListUtils.defaultIfNull(appArgs.getTestSuites(), new ArrayList<String>());
+        for(String testSuite : testSuites){
```

Comment:
@orius123 good idea. Anyway, regarding the default test suite, I still need to have a special treatment, since we want to add it even if it is not specifically written